libavfilter/dnn/dnn_backend_openvino.c

```diff
@@ -329,6 +333,18 @@ static DNNReturnType get_output_ov(void *model, const char *input_name, int inpu
     in_frame->width = input_width;
     in_frame->height = input_height;
 
+    if (ctx->options.input_resizable) {
+        status = ie_network_get_input_shapes(ov_model->network, &input_shapes);
+        input_shapes.shapes->shape.dims[2] = input_height;
+        input_shapes.shapes->shape.dims[3] = input_width;
+        status |= ie_network_reshape(ov_model->network, input_shapes);
+        ie_network_input_shapes_free(&input_shapes);
+        if (status != OK) {
+            av_log(ctx, AV_LOG_ERROR, "Failed to reshape input size for %s\n", input_name);
+            return DNN_ERROR;
+        }
+    }
+
```

Comment:
what happens if this function is not called

Reply:
I didn't get it, under what circumstances will this func not be called?
And if this func not be called, the openvino model and inference request will not be created. So it will report errors in running openvino inference in execute_model_ov, I think.

Reply:
for example filter detect.

Reply:
I moved the model & inference create to init_model_ov, and call model & inference create again in execute model if it's not initiated

Reply:
still see the code here, not in init_model_ov

Reply:
um, this part of code is for input resize. It should be here, in order to be called by get_output_ov, right?
I think it's not reasonable to put it in init_model_ov

Reply:
ok, let's put it here now.